Code/Editor/EditorEngineProcessFramework/EngineProcess/EngineProcessDocumentContext.h

```diff
@@ -15,6 +15,24 @@ class ezExportDocumentMsgToEngine;
 class ezCreateThumbnailMsgToEngine;
 struct ezResourceEvent;
 
+struct ezEngineProcessDocumentContextFlags
+{
+  typedef ezUInt8 StorageType;
+
+  enum Enum
+  {
+    None = 0,
+    CreateWorld = EZ_BIT(0),
+    Default = CreateWorld
```

Comment:
Maybe I would make None the default, to make this more explicit everywhere ?

Reply:
Done